gabrielgomesml/training_project_client
# Feature/discussion-chat

## What I did

- Create a section for users discussions with socket.io

## How to test

- First of all make sure you are running the most updated server version in your machine. You can get it here: https://github.com/gabrielgomesml/training_project_server/pull/3
- Open the project and use the command `nvm use`
- Run a `yarn` or `yarn install`
- After that run the command `yarn start` to start the project
- Log into your account and you'll be redirect to the homepage;
- Click in the "Discussões" button in the navbar;
- Create a nickname to be used in the chat and after that create a new room;
- In another account (i recommend log into it with an anonymous window) do the same proccess and enter in the room that you create.

P.S: I will still improve the UI of the page and add error handles requested in the other PR.
